x-pack/test/cloud_security_posture_api/utils.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RetryService } from '@kbn/ftr-common-functional-services';
+import type { Agent } from 'supertest';
+import type { ToolingLog } from '@kbn/tooling-log';
+import type { Client as EsClient } from '@elastic/elasticsearch';
+import expect from '@kbn/expect';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+
+/**
+ * Checks if plugin initialization was done
+ * Required before indexing findings
+ */
+export const waitForPluginInitialized = ({
```

Comment:
nice enhancement, for the future I guess we can pass only the provider context and retrieve the parameters from it